File: C++/3454.cpp
#include <iostream>

using namespace std;

int main() {
    
    string tabuleiro;
    cin >> tabuleiro;
    
    if((tabuleiro == "XXO") || (tabuleiro == "OXX"))
        cout << "Alice";
    else if (tabuleiro == "XOX")
        cout << "*";
    else
        cout << "?";
    cout << endl;
    
    return 0;
}

File: javascript/3454.js
var input = require('fs').readFileSync('/dev/stdin', 'utf8');
var tabuleiro = input.trim();

if((tabuleiro == "XXO") || (tabuleiro == "OXX"))
    console.log("Alice");
else if (tabuleiro == "XOX")
    console.log("*");
else
    console.log("?");